code/game/objects/items/devices/airlock_painter.dm

```diff
@@ -0,0 +1,61 @@
+// Airlock painter
+
+/obj/item/airlock_painter
+	name = "airlock painter"
+	desc = "An advanced autopainter preprogrammed with several paintjobs for airlocks. Use it on a completed airlock to change its paintjob."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "airlock_painter"
+	item_state = "airlock_painter"
+
+	usesound = 'sound/effects/spray2.ogg'
+
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+
+	materials = list(MAT_METAL = 3000, MAT_GLASS = 1000)
+
+
+/obj/item/airlock_painter/Initialize()
```

Comment:
unnecessary if you're just gonna call . = ..()

Reply:
true that